lms/static/sass/_header.scss

```diff
@@ -0,0 +1,365 @@
+/*
+  This file contains all the necessary styling for a uniform
+  navigation bar that can exist on any of the v1, v2 or bootstrap
+  pages.
+*/
+.global-header {
+  box-sizing: border-box;
+  width: 100%;
+  overflow: hidden;
+  padding-bottom: $baseline/2;
+  border-bottom: 1px solid theme-color("primary");
+  box-shadow: 0 1px 5px 0 rgba(0, 0, 0, 0.1);
+  background: $white;
```

Comment:
Use `theme-color("inverse")`. I don't know that we'll ever support dark themes, but I'd like to try to support it.